src/caselawclient/models/documents/__init__.py

```diff
@@ -329,31 +329,39 @@ def status(self) -> str:
 
         return DOCUMENT_STATUS_NEW
 
-    def force_enrich(self) -> None:
+    def force_enrich(self) -> bool:
         """
-        Request enrichment of the document, but do no checks
+        Request enrichment of the document, regardless of time checks,
+        but still making sure it's sensible to do so.
         """
         now = datetime.datetime.now(datetime.timezone.utc)
+        # We save the time before checking for docx so if there isn't a
+        # docx we will skip this document
         self.api_client.set_property(
             self.uri,
             "last_sent_to_enrichment",
             now.isoformat(),
         )
 
+        if not (self.can_enrich):
+            print("Enrichment of {self} rejected")
```

Comment:
This feels like it should be an exception. Something is trying to force an enrichment when it shouldn't, and we have a method in `can_enrich` to check if it should be possible beforehand.